test/integration/spec/connect.js

```diff
@@ -805,7 +805,7 @@ describe('connect', function() {
         // TODO: Remove firefox check once this firefox bug is fixed
         // https://bugzilla.mozilla.org/show_bug.cgi?id=1688342
         if (!isFirefox) {
-          it(`should ${maxBitrates[kind] ? '' : 'not '}limit the ${kind} bitrate`, () => {
+          it(`should ${maxBitrates[kind] ? '' : 'not '}limit the ${kind} bitrate (@unstable: JSDK-2512)`, () => {
```

Comment:
Please file a new bug to go with @unstable 
This bug should be about fixing this test to make it stable. I think there is also an epic to go with unstable test bugs.